### DISCUSSION TITLE
Google Chrome Exatensions.

### DISCUSSION THREAD

**Discussion by mufasi4** [SafeExamBrowser/seb-win-refactoring#609] [Q&A]
I am writing this to enquire whether there is any method to allow **GOOGLE CHROME EXTENSIONS** which are available in the google chrome store.

As we will be prohibiting the usage of the GOOGLE Chrome Application during the examination process, Is there any option to Allow the usage of Google Chrome Extensions which are available in the Chrome store during the examination?

Please provide your valuable opinion on this query.

Thank You.

**Comment by dbuechel** [comment]
Most unfortunately, there is currently no option to allow Google Chrome extensions within SEB. And as we're using Chromium as browser engine, and not Google Chrome itself, this most likely will remain so for the time being.
